examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -43,15 +43,17 @@
 .. include:: ../../../examples/masker_note.rst
 
 """
+import matplotlib.pyplot as plt
```

Comment:
Same, I think we want this import below the comment

Reply:
done